ncm-spma/src/main/perl/spma/yumdnf.pm

```diff
@@ -8,6 +8,7 @@ use Set::Scalar;
 push our @ISA , qw(NCM::Component::spma::yum);
 
 use constant YUM_CONF_FILE => "/etc/dnf/dnf.conf";
+use constant YUM_CONF_CLEANUP_ON_REMOVE_VALUE => 0;
```

Comment:
Why not?

Reply:
this is potentionally dangerous; and i was surprised it was enabled on yum.

the usual scenario is that a (installed) leave has a  (installed) dependency on something you have listed in the packages list (but the leave itself is not listed; only the dependency). the current code will generate a remove command for the leave, and no install command for the dependency (it's already nstalled). autoremove will remove the leave and the dependency; and you will end up with something you didn't ask for.

Reply:
OK, makes sense - thanks,